src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoSubmissionServiceImpl.java

```diff
@@ -305,5 +308,24 @@ private XMLGregorianCalendar getComputedSubmittedDate(String location) {
         }
 
     }
+    //This function will be used when cso has the valid flag available
+    private Boolean validateJson(Object json) {
+
+        if (json == null) return false;
+        logger.info("Validate form data");
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            mapper.enable(DeserializationFeature.FAIL_ON_READING_DUP_TREE_KEY);
+            JsonFactory factory = mapper.getFactory();
+            JsonParser parser = factory.createParser(json.toString());
+            mapper.readTree(parser);
+            logger.info("Form data is valid");
+            return true;
+        } catch (Exception ex) {
+            logger.info("Form data is invalid");
```

Comment:
is this an info or an error? Do we need to pass anything else in the log message here?